lld/docs/ReleaseNotes.rst

```diff
@@ -41,9 +41,25 @@ COFF Improvements
 * Added support for ``--time-trace`` and associated ``--time-trace-granularity``.
   This generates a .json profile trace of the linker execution.
 
+* Prefer library paths specified with ``-libpath:`` over the implicitly
```

Comment:
This should be clarified. Is this telling me the reader that I should use -libpath instead of letting it be implicit?

Seems like it's missing a noun like "<tool> now prefers ....".